apps/books/templates/base.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>{% block title %}{% endblock title%}</title>
+    </head>
+    <body>
+        {% include "header.html" %}
+        <div id="content">
+            {% block content %}
+            {% endblock content %}
+        </div>
+        {% include "footer.html" %}
+    </body>
+</html>
```

Comment:
Is `base.html` is only for books app? If not please move it to some generic/common place ..

Reply:
May be you can add templates on root level and place common templates there ..

Reply:
Done